src/redismodule.h

```diff
@@ -1000,6 +1000,8 @@ REDISMODULE_API void (*RedisModule_ChannelAtPosWithFlags)(RedisModuleCtx *ctx, i
 REDISMODULE_API unsigned long long (*RedisModule_GetClientId)(RedisModuleCtx *ctx) REDISMODULE_ATTR;
 REDISMODULE_API RedisModuleString * (*RedisModule_GetClientUserNameById)(RedisModuleCtx *ctx, uint64_t id) REDISMODULE_ATTR;
 REDISMODULE_API int (*RedisModule_GetClientInfoById)(void *ci, uint64_t id) REDISMODULE_ATTR;
+REDISMODULE_API int (*RedisModule_SetClientNameById)(uint64_t id, RedisModuleString *name) REDISMODULE_ATTR;
+REDISMODULE_API int (*RedisModule_SetClientProtocolById)(uint64_t id, int proto) REDISMODULE_ATTR;
```

Comment:
it seems odd to me that we have setters for these, but no getters.
i see that RM_GetClientInfoById doesn't return that INFO.
furthermore, it seems odd to me that we can even set these (specifically the RESP version), for an arbitrary client.
maybe instead we should have these setters only work on the current client of the module context?
then for the RESP version, we have a "getter" in RM_GetContextFlags.